dissect/target/plugins/os/unix/ssh.py

```diff
@@ -34,12 +36,26 @@
 PrivateKeyRecord = TargetRecordDescriptor(
     "unix/ssh/private_key",
     [
+        ("datetime", "ts"),
```

Comment:
```suggestion
        ("datetime", "mtime_ts"),
```
Just to be more clear.